static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
```

Comment:
OK so I think I can see a way to make this work, but it's going to be a messy hack. the input I'm using is not meant to be used this way at all - so I'm going to have to chop out the country code in the `onChange` handler, record it separately, and then pass the value of pasting them together in as the value to the input. Which do you think would be easier to change, how it's done on the python side of things or here?